webvtt/parsers.py

```diff
@@ -218,6 +234,15 @@ def _validate(self, lines):
         if not re.match('WEBVTT', lines[0]):
             raise MalformedFileError('The file does not have a valid format')
 
+    def _is_empty(self, block):
```

Comment:
We could have this method as one liner:`return not any(line.strip() for line in block.lines)`

Reply:
Do you prefer the oneliner style?  I find one liners less readable in general, so I try to make the code cleaner.  But I can change this if you feel strongly about it one way or the other